valheim_plus.cfg

```diff
@@ -140,6 +140,9 @@ enabled=false
 ; Allows for - values to shorten food duration.
 foodDurationMultiplier=0
 
+; If set to false, this option prevents food degrading over time - in other words, it retains its maximum benefit until it expires.
+foodDegradesOverTime=true
```

Comment:
I think since a majority of our bool configurations default to false, we should continue that here to prevent user confusion and rename this to `disableFoodDegradation=false` or something similar.